src/util/memory_limit.h

```diff
@@ -1,18 +0,0 @@
-/*******************************************************************\
-
-Module:
-
-Author: Peter Schrammel, peter.schrammel@diffblue.com
-
-\*******************************************************************/
-
-#ifndef CPROVER_UTIL_MEMORY_LIMIT_H
-#define CPROVER_UTIL_MEMORY_LIMIT_H
```

Comment:
This is used by TG. We can move it there if you want to remove it from this repo.

Reply:
The code isn't portable so it might make sense to move it? I don't have very strong feelings about it, but given that the code can't be tested in this repository I'd prefer to move it to a place where it _is_ being tested.

Reply:
Moved to #2214.